app/views/content/funding-and-support/if-youre-a-veteran.md

```diff
@@ -4,9 +4,6 @@ heading: "Funding and support if you're a veteran"
 subcategory: Extra support
 description: |-
     Find out how to get support training to teach if you're a veteran transitioning from or you've already left the armed forces.
-related_content:
-    Explore life as a teacher: "/life-as-a-teacher"
-    Teach in further education without a degree: "https://www.teachinfurthereducation.education.gov.uk/"
```

Comment:
I wonder if we should include an FE and EY section on this page (as @emmaachesongray did on the If you don't have a degree page) in the interests of cross-workforce support?

But if it doesn't work, let's discuss!